package.json

```diff
@@ -13,4 +13,3 @@
     "morgan": "~1.9.0",
     "nodemon": "^2.0.2"
   }
-}
```

Comment:
Careful with stuff like this. Since this was merged into master, the repo was broken and would not build.